### ISSUE TITLE
Aggressive Expansion

### ISSUE THREAD

**Issue by jamesgraves** [jamesgraves/bonzAI#1]
This issue may eventually be submitted upstream to http://github.com/bonzaiferroni/bonzAI

Create an option for more aggressive expansion.  As soon as reasonable, start mining adjacent rooms for energy.

Also, as soon as GCL level gets to 2 or higher, have an option to start claiming another room.

**Comment by bonzaiferroni** [comment]
Yeah this is a good idea, the main thing that was stopping me is being able to spawn a claimer that is big enough to keep a reservation. But then I realized, we could just use two half size claimers. Since I have less extra time to spend on screeps recently there is quite a backlog of things to implement :) looking forward to seeing what you do with your fork

**Comment by jamesgraves** [comment]
OK, I'll look at that.  I'm still just starting out with TypeScript and Javascript though.
